### ISSUE TITLE
bundle create command deletes manifests contents

### ISSUE THREAD

**Issue by cliveseldon** [operator-framework/operator-sdk#3079]
## Bug Report

**What did you do?**
Followed: https://sdk.operatorframework.io/docs/olm-integration/user-guide/
`operator-sdk generate csv --csv-version 0.6.0`
` operator-sdk bundle create --generate-only`

**What did you expect to see?**
The manifests and metadata to be created 

**What did you see instead? Under which circumstances?**
The manifest files were zeroed. They exists but have been cleared - 0 bytes.

**Environment**
* operator-sdk version: v0.17.0

* go version: go1.13.10 linux/amd64

* Kubernetes version information:

* Kubernetes cluster kind: 

* Are you writing your operator in ansible, helm, or go?

**Possible Solution**

**Additional context**

**Comment by camilamacedo86** [comment]
c/c @estroz

**Comment by cliveseldon** [comment]
Running:
```
operator-sdk bundle create --generate-only --directory ./deploy/olm-catalog/seldon-deploy-operator/0.6.0/
```

sort of works - it creates a new manifests and metadata folders alongside an existing 0.6.0 folder copying the manifests from 0.6.0.

Its not clear whether package or manifest structure is the current way forward and how these commands work with these unless I am missing something?

**Comment by estroz** [comment]
@cliveseldon I believe this was fixed in v0.17.1, see #2879.
